src/test/java/apoc/es/ElasticSearchTest.java

```diff
@@ -226,34 +242,14 @@ public void testFullSearchWithQueryAsAStringWithEquals() throws Exception {
      */
     @Test
     public void testFullSearchWithOtherParametersAsAString() throws Exception {
-        TestUtil.testCall(db, "CALL apoc.es.query({host},{index},{type},'size=1&scroll=1m&_source=true',null) yield value", defaultParams, r -> {
+        TestUtil.testCall(db, "CALL apoc.es.query({host},{index},{type},'size=1&scroll=1m&_source=true&q=procedureName:get',null) yield value", defaultParams, r -> {
             Object hits = extractValueFromResponse(r, "$.hits.hits");
             assertEquals(1, ((List) hits).size());
             Object name = extractValueFromResponse(r, "$.hits.hits[0]._source.procedureName");
             assertEquals("get", name);
         });
     }
 
-    /**
```

Comment:
where did this go?

Reply:
The test was wrong it had 6 arguments instead of 5 required by the `apoc.es.post`, we already test the add of new documents in `insertDocuments`. I can create a new one if you want.